src/styles/blocks/header.scss

```diff
@@ -0,0 +1,10 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
```

Comment:
It's a good practice to use variables for colors and box shadows as well, which will make it easier to update them in the future if needed. Define a variable for the background color and the box shadow and use them here.